prime-router/src/main/kotlin/encoding/FHIR.kt

```diff
@@ -0,0 +1,74 @@
+package gov.cdc.prime.router.encoding
+
+import ca.uhn.hl7v2.model.Message
+import ca.uhn.hl7v2.model.v251.segment.MSH
+import io.github.linuxforhealth.fhir.FHIRContext
+import io.github.linuxforhealth.hl7.ConverterOptions
+import io.github.linuxforhealth.hl7.message.HL7MessageEngine
+import io.github.linuxforhealth.hl7.resource.ResourceReader
+import org.apache.logging.log4j.kotlin.Logging
+import org.hl7.fhir.r4.model.Bundle
+
+fun Bundle.encode(): String {
+    return FHIR.encode(this)
+}
+
+class FHIR {
+    companion object : Logging {
+        val resourceReader = ResourceReader.getInstance()
+
+        val messagetemplates = resourceReader.getMessageTemplates()
```

Comment:
Small case of pedantry: Kotlin style uses camelCase. Should be `messageTemplates`.